surfstat/python/SurfStatResels.py

```diff
@@ -392,5 +396,7 @@ def py_SurfStatResels(slm, mask=None):
     lkcs = np.atleast_2d(lkcs)
     D = lkcs.shape[1]-1
     resels = lkcs / np.sqrt(4*np.log(2))**np.arange(0,D+1)
+    # back to matlab indexing by +1, otherwise tests will fail
+    edg = (edg + 1).astype(int)
```

Comment:
Same as the previous comment; whilst this matches MATLAB exactly it doesn't make sense from an end-user perspective i.e. there's no reason within Python to start counting from 0. Any usage of this output argument will always start with `edg = edg-1`

Reply:
quick answer to both comments: 
when edg and reselspvert from SurfStatResels.py must be identical to that of SurfStatResels.m.
Otherwise SurfStatP.py does not work properly (since it is calling SurfStatResels.py internally).

Let's keep everything identical at first stage to matlab, otherwise we have to always make *tricks* while testing things.
Then, we can discuss later if we keep edg starting from 1, or our mask arrays as bool or int, etc.